hack/fetch-official-cve-feed.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import sys
+import requests
+
+#print("\nToken:", sys.argv[1])
+
+url = 'https://api.github.com/search/issues?q=is:issue+CVSS:3+in:body+label:committee/\
+security-response+state:closed+repo:kubernetes/kubernetes'
+headers = {'Authorization': f"token {sys.argv[1]}",\
```

Comment:
Why do we need a token for this ? I can interrogate this route without a token. 

```
 curl -I https://api.github.com/search/issues\?q\=is:issue+CVSS:3+in:body+label:committee/\                                                                                                                                      02:49:15 PM
security-response+state:closed+repo:kubernetes/kubernetes
HTTP/2 200
server: GitHub.com
date: Mon, 25 Jul 2022 16:03:26 GMT
content-type: application/json; charset=utf-8
cache-control: no-cache
vary: Accept, Accept-Encoding, Accept, X-Requested-With
x-github-media-type: github.v3; format=json
access-control-expose-headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
access-control-allow-origin: *
strict-transport-security: max-age=31536000; includeSubdomains; preload
x-frame-options: deny
x-content-type-options: nosniff
x-xss-protection: 0
referrer-policy: origin-when-cross-origin, strict-origin-when-cross-origin
content-security-policy: default-src 'none'
x-ratelimit-limit: 10
x-ratelimit-remaining: 9
x-ratelimit-reset: 1658765066
x-ratelimit-resource: search
x-ratelimit-used: 1
accept-ranges: bytes
x-github-request-id: D812:93C8:5FD106:61CF7F:62DEBECE
```

Reply:
Yes can confirm the same. I think we would need token if we hit rate-limits. But the current rate limits for unauthenticated users should be okay for our usecase: https://docs.github.com/en/rest/search#rate-limit

Reply:
Updated